tests/test_cfg/snapshots/sample_cfg_output.txt

```diff
@@ -0,0 +1,25 @@
+digraph my_file {
```

Comment:
Good! Usually for snapshot tests the convention is to name the file analogously to the input. So here that might be `my_file.gv` (see <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>)